client/src/components/ElementEditor/Element.js

```diff
@@ -66,11 +86,16 @@ class Element extends Component {
           title={Title}
           elementType={BlockSchema.type}
           fontIcon={BlockSchema.iconClass}
+          link={link}
+          caretClickCallback={(event) => this.handleExpand(event)}
```

Comment:
`handleExpand` is already bound to `this` in the constructor - I think you could use `caretClickCallback={this.handleExpand}` here